clients/client-python/tests/integration/integration_test_env.py

```diff
@@ -17,7 +17,7 @@
 
 def get_gravitino_server_version():
     try:
-        response = requests.get("http://localhost:8090/api/version")
+        response = requests.get("http://localhost:8090/api/version", timeout=10)
```

Comment:
Hi @adheeshsharma19, thanks for your contribution.

There are some problems that you may need to fix:
1. It's not the best practice that putting a number directly in the file, I would suggest that you can create a file to store constant values like `TIMEOUT = 10` and import it in the files.
2. Can you explain why the timeout should be 10 ?